agents/pfeiffer_tpg366/pfeiffer_tpg366_agent.py

```diff
@@ -0,0 +1,50 @@
+#script to log and readout pfeiffer TPG 366 gauge contoller
+#via Ethernet connection
+#Zhilei X., Tanay B.
+
+import socket
+import numpy as np
+
+BUFF_SIZE = 128
+ENQ = '\x05'
+IP_ADDRESS = '10.10.10.20'
```

Comment:
This works for now, but when we add the OCS Agent code we'll want to make this a command line option which will become configuration within the ocs configuration file, that way other devices can be used with the same code either in the same network, or elsewhere.

Reply:
I guess the fix will become obvious when the entire agent is witten-up, I will wait to address this later

Reply:
This comment still applies, for others to use this Agent this'll have to not be hardcoded.